### PR TITLE
Multisig refactor

### PR DESCRIPTION
**Pull request by anakin001**
Edit Multisig refactor and updated functions on transfer page

### CODE REVIEW

File: src/modules/account/submodule/multisig/views/ViewAccountMultisigEditAccount.vue [hunk 1]
```diff
@@ -184,35 +185,47 @@ export default {
...
     // current wallet
     const wallet = walletState.currentLoggedInWallet;
-
     // get account details initialization
-    const acc =  walletState.currentLoggedInWallet.accounts.find(element => element.name ===p.name);
-    const cosigneraddress = cosigner => Address.createFromPublicKey(cosigner, networkState.currentNetworkProfile.network.type).plain().substr(-4)
+    const acc =  computed(()=>{
+      let account
+      if (wallet.accounts.find(element => element.name ===p.name) === undefined && wallet.others.find(element => element.name ===p.name) === undefined){
+        return new WalletAccount("a","b","c","d","e","f")
```
Comment (sleepyOwl14) [inline src/modules/account/submodule/multisig/views/ViewAccountMultisigEditAccount.vue:200]: what does this code do ? any reason to return it ?

Comment (anakin001) [reply src/modules/account/submodule/multisig/views/ViewAccountMultisigEditAccount.vue:200]: when i refresh this page, there are chances that both acc goes undefined, i need this validation to get rid of the error

Comment (sleepyOwl14) [reply src/modules/account/submodule/multisig/views/ViewAccountMultisigEditAccount.vue:200]: I mean why return a fake WalletAccount, why not just return null ?

**Comment by sleepyOwl14** [comment]
can add in description for PR next time ? It is multisig, but need to specify on what. It involve both multisig edit page and transfer page.
If the title itself is sufficient and clear then ok

File: src/modules/account/submodule/multisig/views/ViewAccountMultisigEditAccount.vue (hunk 1 above)
Comment (anakin001) [reply src/modules/account/submodule/multisig/views/ViewAccountMultisigEditAccount.vue:200]: fixed on my end

**Review by sleepyOwl14 — APPROVED** [review]

**Pull request merged**